libs/packages/components/src/lib/toolbar/toolbar.component.ts

```diff
@@ -64,7 +64,7 @@ export class SdsToolbarComponent {
 
   // Elements that surround the toolbar need to make space for the expanded toolbar.
   // Adding 20 more pixes to provide padding
-  @Input() expandedSpace: string = "320px";
+  @Input() expandedSpace: string = "20px";
```

Comment:
Because "absolute" position has been removed - we don't need this input or the directive that is associated with the expand animation - please remove.